packages/common/src/extensions/slickCellExternalCopyManager.ts

```diff
@@ -157,7 +157,9 @@ export class SlickCellExternalCopyManager {
   setDataItemValueForColumn(item: any, columnDef: Column, value: any): any | void {
     if (!columnDef?.denyPaste) {
       if (this._addonOptions.dataItemColumnValueSetter) {
-        return this._addonOptions.dataItemColumnValueSetter(item, columnDef, value);
+        if (this._addonOptions.dataItemColumnValueSetter(item, columnDef, value) !== true) {
```

Comment:
wow triple `if`, can we merge some of them? 😄

Reply:
absolutely. what do you think about the change in general? its kind of a small breaking change but I dont assume anyone returned true so far

Reply:
I searched the internet for SlickGrid and `dataItemColumnValueSetter` and couldn't find anything, I searched my own examples and I don't have any demo with it. So I seriously don't think anyone uses apart from you maybe haha. But what does it mean to "Return true if default logic should continue to evaluate"? Continue evaluating what and where? Do you have a demo of this?

I searched on SlickGrid core lib and found 1 example with it, the code is like below and the live demo is this [Spreadsheet: features of the previous example but using a DataView](http://6pac.github.io/SlickGrid/examples/example-excel-compatible-spreadsheet-dataview.html)
```ts
  function getVal(item, columnDef) {
    //return dataView.getItemById(item.id)[columnDef.field];
    return item[columnDef.field];
  }

  function setVal(item, columnDef, value) {
    item[columnDef.field] = value;
//    dataView.updateItem(item.id, item);
  }

  var options = {
    editable: true,
    enableAddRow: true,
    enableCellNavigation: true,
    asyncEditorLoading: false,
    autoEdit: false,
    dataItemColumnValueExtractor: getVal,
    dataItemColumnValueSetter: setVal
  };
```

Reply:
do you mean `copyActiveEditorCell: true`? because I don't see any usage of `dataItemColumnValueSetter`

Reply:
no the last commit `if (this._addonOptions.dataItemColumnValueSetter(item, columnDef, value) !== true) {
`

so if that now returns true the rest if the original method will continue, whereas before it just exited with the return